src/agent/codecoveragepublisher.ts

```diff
@@ -0,0 +1,233 @@
+import testifm = require('vso-node-api/interfaces/TestInterfaces');
+import ctxm = require('./context');
+import cm = require('./common');
+import utilities = require('./utilities');
```

Comment:
top level concern I have is that the main implementation is in vso-agent. when we move to common impl for CC publish how will window agent pull these changes onto windows box.